services/clusters-keeper/src/simcore_service_clusters_keeper/core/settings.py

```diff
@@ -12,21 +12,20 @@
 from pydantic import Field, NonNegativeInt, PositiveInt, parse_obj_as, validator
 from settings_library.base import BaseCustomSettings
 from settings_library.docker_registry import RegistrySettings
+from settings_library.ec2 import EC2Settings
 from settings_library.rabbit import RabbitSettings
 from settings_library.redis import RedisSettings
 from settings_library.utils_logging import MixinLoggingSettings
 from types_aiobotocore_ec2.literals import InstanceTypeType
 
 from .._meta import API_VERSION, API_VTAG, APP_NAME
 
+CLUSTERS_KEEPER_ENV_PREFIX: Final[str] = "CLUSTERS_KEEPER_"
 
-class EC2ClustersKeeperSettings(BaseCustomSettings):
-    EC2_CLUSTERS_KEEPER_ACCESS_KEY_ID: str
-    EC2_CLUSTERS_KEEPER_ENDPOINT: str | None = Field(
-        default=None, description="do not define if using standard AWS"
-    )
-    EC2_CLUSTERS_KEEPER_REGION_NAME: str = "us-east-1"
-    EC2_CLUSTERS_KEEPER_SECRET_ACCESS_KEY: str
+
+class ClustersKeeperEC2Settings(EC2Settings):
+    class Config:
+        env_prefix = CLUSTERS_KEEPER_ENV_PREFIX
```

Comment:
MINOR: what about adding a little example of these settings. 

BTW: shouldnt' you inherit from `Ec2Settings.Config`. Take into account that the base class CustomSettings has a Config!

Reply:
good point